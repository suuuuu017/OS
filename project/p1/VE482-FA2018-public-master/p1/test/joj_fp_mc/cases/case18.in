WRITELN echo 123 > 1.txt
WRITELN echo 321 > 2.txt
WRITELN echo abc < 1.txt < 2.txt
WRITELN echo abc | grep abc < 1.txt
WRITELN exit
