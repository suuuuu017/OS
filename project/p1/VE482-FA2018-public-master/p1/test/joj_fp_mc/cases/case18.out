mumsh $ mumsh $ mumsh $ error: duplicated input redirection
mumsh $ error: duplicated input redirection
mumsh $ exit
